src/util/amount.rs

```diff
@@ -914,12 +924,50 @@ pub mod serde {
         }
     }
 
+    pub fn parse<'d, D, A>(d: D) -> Result<A, D::Error>
+    where
+        D: Deserializer<'d>,
+        A: SerdeAmount,
+        <A as FromStr>::Err: ::std::fmt::Display,
+    {
+        use ::std::fmt::{self, Display, Formatter};
+        use ::std::marker::PhantomData;
+
+        struct Visitor<ValueT>(PhantomData<ValueT>);
+
+        impl<'de, ValueT> serde::de::Visitor<'de> for Visitor<ValueT>
```

Comment:
Fixes 1.29 compilation error
```suggestion
        impl<'de, ValueT> ::serde::de::Visitor<'de> for Visitor<ValueT>
```